bench_mystr_test.go

```diff
@@ -0,0 +1,28 @@
+// Import StrToInt1 and do Benchmark tests
+package StrToInt2
```

Comment:
Please name packages with respect to standards https://golang.org/doc/effective_go.html#names